warehouse/models/mart/gtfs/_mart_gtfs.yml

```diff
@@ -1240,3 +1240,9 @@ models:
           - relationships:
               to: ref('dim_schedule_feeds')
               field: key
+  - name: fct_daily_all_feeds
+    description: |
+      Unioned model of Schedule and RT daily feeds.
```

Comment:
Right, I think the concept of a daily RT feed is ambiguous and one of the reasons why I don't think the "daily" models are super appropriate for use outside of validations. 🤔

Reply:
I'm trying to understand this back and forth - I'm not sure I understand since we also have fact daily tables for schedule and RT feeds. Could you clarify what the purpose of this unioned table is?

Reply:
Yeah I think I am going to remove this; I had a use case in mind but it didn't up being relevant.